### PR TITLE
feature: added VC and VP format

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: packages/ssi-types/src/types/vc.ts [hunk 1]
```diff
@@ -195,6 +195,8 @@ export interface JwtDecodedVerifiablePresentation {
 }
 
 export type ClaimFormat = 'jwt' | 'jwt_vc' | 'jwt_vp' | 'ldp' | 'ldp_vc' | 'ldp_vp' | string
+export type CredentialFormat = 'jwt' | 'ldp' | string
```
Comment (nklomp) [inline packages/ssi-types/src/types/vc.ts:198]:
```suggestion
export type CredentialFormat = 'jwt' | 'ldp' | 'jwt_vc' | 'ldp_vc' | string
```

File: packages/ssi-types/src/types/vc.ts [hunk 2]
```diff
@@ -195,6 +195,8 @@ export interface JwtDecodedVerifiablePresentation {
 }
 
 export type ClaimFormat = 'jwt' | 'jwt_vc' | 'jwt_vp' | 'ldp' | 'ldp_vc' | 'ldp_vp' | string
```
Comment (nklomp) [inline packages/ssi-types/src/types/vc.ts:197]:
```suggestion
export type ClaimFormat = CredentialFormat | ClaimFormat
```

Comment (nklomp) [reply packages/ssi-types/src/types/vc.ts:197]: Needs to be moved below the other properties probably

**Review by nklomp — CHANGES_REQUESTED** [review]
See suggestions

File: packages/ssi-types/src/types/vc.ts (hunk 2 above)
Comment (sksadjad) [reply packages/ssi-types/src/types/vc.ts]: moved claim lower than `CredentialFormat` and `PresentationFormat`

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**